pdl_schema.md

```diff
@@ -0,0 +1,1106 @@
+deprecated types will be marked as deprecated.
+
+It is recommended to specify a string describing the reason for deprecation and
+the alternative as the value for the `@deprecated` annotation.
+
+Deprecate a field:
+```pdl
+namespace com.example.models
+
+import com.example.time.Date
+
+record User {
+  firstName: string
+
+  @deprecated = "Use birthday instead."
+  birthYear: int
+
+  birthday: Date  
+}
+```
+Deprecate a record:
+```pdl
+namespace com.example.models
+
+@deprecated = "Use Person type instead."
+record User {
+  firstName: string
+}
+```
+
+See [Enum documentation and deprecation](#enum-documentation-and-deprecation)
+for details on deprecating enum symbols.
+
+### Including fields
+Pegasus records support including fields from one or more other records. When
+a record is included, all its fields will be included in the current record. It
+does not include any other attribute of the other record.
+
+Includes are transitive, if record A includes record B and record B includes
+record C, record A contains all the fields declared in record A, record B and
+record C.
+
+The value of the “include” attribute should be a list of records or typerefs of
+records. It is an error to specify non-record types in this list.
+
+For example:
+```pdl
+namespace com.example.models
+
+/**
+ * User includes fields of AuditStamp, User will have fields firstName from
+ * itself and fields createdAt and updatedAt from AuditStamp.
+ */
+record User includes AuditStamp {
+  firstName: string
+}
+```
+
+```pdl
+namespace com.example.models
+
+/**
+ * A common record to represent audit stamps.
+ */
+record AuditStamp {
+  /** Time in milliseconds since epoch when this record was created */
+  createdAt: long
+
+  /** Time in milliseconds since epoch when this record was last updated */
+  updatedAt: long
+}
+```
+
+Includes feature allows including fields from multiple records.
+
+For example:
+```pdl
+namespace com.example.models
+
+/** A common record for specifying version tags of a record */
+record VersionTag {
+  versionTag: string
+}
+```
+
+```pdl
+namespace com.example.models
+
+/**
+ * User includes fields of AuditStamp, User will have fields firstName from
+ * itself and fields createdAt and updatedAt from AuditStamp.
+ */
+record User includes AuditStamp, VersionTag {
+  firstName: string
+}
+```
+
+***Note:*** Record inclusion does not imply inheritance, it is merely a
+convenience to reduce duplication when writing schemas.
+
+
+## Primitive Types
+The above examples already introduced `int`, `long`, `string` and `boolean`
+primitive types that are supported by Pegasus.
+The full list of supported Pegasus primitive types are: `int`, `long`, `float`,
+`double`, `boolean`, `string` and `bytes`.
+
+The actual types used for the primitives depends on the language specific
+binding implementation. For details on Java bindings for Pegasus primitives, see
+[Primitive Types] (https://linkedin.github.io/rest.li/java_binding#primitive-types)
```

Comment:
This link isn't rendering properly. I think it's the space between the two components.